acceptance/files/complex-output

```diff
@@ -0,0 +1,50 @@
+Defined with arguments:
+* ArgArray of type array
+* ArgArrayOfHashes of type hashtable[]
+* ArgBool of type bool
+* ArgDateTime of type datetime
+* ArgFileInfo of type System.IO.FileInfo
+* ArgFloat of type float
+* ArgGuid of type guid
+* ArgHashtable of type hashtable
+* ArgInt32 of type int
+* ArgRegex of type regex
+* ArgString of type string
+* ArgSwitch of type switch
+* ArgTimeSpan of type timespan
+
+Received arguments:
+* ArgArray (System.Object[]):
+0
+foo
+True
+key: value
+* ArgArrayOfHashes (hashtable[]):
+name: puppet
+version_requirement: >= 3.7.0 < 6.0.0
+name: pe
+version_requirement: >= 5.0.0
+* ArgBool (bool):
+False
+* ArgDateTime (datetime):
+9/8/2017 2:23:59 PM
```

Comment:
This might need some munging eventually, on PowerShell 6 on my Mac this was printing `9/8/17 2:23:59 PM`.